src/plugins/ni/testmod_ni.c

```diff
@@ -449,12 +449,16 @@ int main (int argc, char ** argv)
 
 	TEST (ver);
 	TEST (new);
+#ifndef __SANITIZE_ADDRESS__
```

Comment:
Sorry, wrong file. The issue is in the [hash.c on line 413](https://blob.libelektra.org/src/plugins/ni/nickel-1.1.0/src/hash.c#L413). There is already a dirty fix for the issue [on line 363](https://blob.libelektra.org/src/plugins/ni/nickel-1.1.0/src/hash.c#L363) for valgrind, which I think I can complement with the __SANITIZE_ADDRESS__ macro. Then the fix is really at the source.